iOS14UIKitTour/iOS14UIKitTour/ModelControllers/ModelController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ModelController.swift
+//  iOS14UIKitTour
+//
+
+import Foundation
+import Combine
+
+class ModelController: ObservableObject {
+    @Published var repositories = [Repository]()
+    
+    var cancellables = Set<AnyCancellable>()
+    
+    init() {
+        fetchRepos()
+    }
+    
+    func fetchRepos() {
+        GithubAPI.fetch(.repositories)
+            .sink(receiveCompletion: { _ in }) { (repositories: [Repository]) in
```

Comment:
When you reference self in a closure always capture `weak self` to avoid memory leaks.
```suggestion
            .sink { [weak self] repositories in
```